server/src/hardware/decoder/decoderfunction.cpp

```diff
@@ -46,10 +46,15 @@ DecoderFunction::DecoderFunction(Decoder& decoder, uint8_t _number) :
     }},
   function{this, "function", DecoderFunctionFunction::Generic, PropertyFlags::ReadWrite | PropertyFlags::Store},
   value{this, "value", false, PropertyFlags::ReadWrite | PropertyFlags::StoreState,
-    [this](bool /*newValue*/)
+    [this](bool newValue)
     {
+      if(timeoutSeconds.value() > 0 && newValue)
```

Comment:
Check here for `type == Momentary || Hold`? For other functions types the timeout value should not have effect.

Reply:
Timeout is reset to zero for all other function types but maybe it's more readable to check type instead

Reply:
Yeah, I prefer to check type, the we don't need to alter the timeout, if someone accidentally changes the type it keeps the timeout value when set back.

Reply:
Done.